client/src/components/Header.js

```diff
@@ -62,22 +59,30 @@ const HeaderWrapper = styled.header`
   align-items: center;
   text-align: center;
 
-  @media only screen and (max-width: 600px) {
-    h1 {
-      font-size: 1.8rem;
-      flex-grow: 2;
-      margin: 0;
-    }
-  }
   > button {
     background: none;
     border: none;
+    padding: 0;
+    display: flex;
+    align-items: center;
   }
   img {
     margin: 0 25px;
     height: 45px;
+    width: 45px;
   }
```

Comment:
For a better responsive experience, try to use relative values like `rem` or `em`.

Reply:
Nice. looks much better. Thanks :)